.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     if: ${{ github.event_name == 'pull_request' }}
     steps:
       - name: Create k8s Kind Cluster
-        uses: helm/kind-action@v1.10.0
+        uses: helm/kind-action@v1.12.0
```

Comment:
What are the changes, and will they break the existing GH workflows?

Reply:
The changes are specified here: https://github.com/helm/kind-action/releases

They do not break existing functionalities.